client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -295,23 +318,54 @@ const PlansSetup = React.createClass( {
 	},
 
 	renderSuccess() {
+		let noticeText;
 		const site = this.props.selectedSite;
 		if ( ! this.props.hasRequested || ! this.props.isFinished ) {
 			return null;
 		}
 
-		const pluginsWithErrors = filter( this.props.plugins, ( item ) => {
+		const plugins = this.addWporgDataToPlugins( this.props.plugins );
+		const pluginsWithErrors = filter( plugins, ( item ) => {
 			return ( item.error !== null );
 		} );
 
 		if ( pluginsWithErrors.length ) {
```

Comment:
Maybe all the code within this `if` block could be extracted to a new `renderErrorMessages` method so it's just 
```
if ( pluginsWithErrors.length ) {
   return this.renderErrorMessages();
}
```

Not that it adds anything, but it would make the code a little more compact